qcut/.agents/skills/libtv-skill/scripts/create_session.py

```
@@ -0,0 +1,64 @@
+        "--session-id",
+        default="",
+        help="已有会话 ID，不传则创建新会话或返回已有默认会话",
+    )
+    args = parser.parse_args()
+
+    data = create_session(session_id=args.session_id or "", message=args.message or "")
+    project_uuid = data.get("projectUuid", "")
+    session_id = data.get("sessionId", "")
+
+    if not session_id:
+        print("错误：未返回 sessionId", file=sys.stderr)
```

Ruff (0.15.13)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)